v3_async_wip/v3_async_wip/mqtt_client.py

```diff
@@ -13,11 +13,30 @@
 
 logger = logging.getLogger(__name__)
 
-OP_TYPE_CONNECT = "CONNECT"
-OP_TYPE_DISCONNECT = "DISCONNECT"
 
-RECONNECT_MODE_DROP = "RECONNECT_DROP"
-RECONNECT_MODE_ALL = "RECONNECT_ALL"
+# NOTE: Paho can return a lot of rc values. However, most of them shouldn't happen.
+# Here are the ones that we can expect for each method.
+expected_connect_rc = [mqtt.MQTT_ERR_SUCCESS]
```

Comment:
>expected_connect_rc [](http://example.com/codeflow?start=0&length=19)

Are these here for documentation only or do they serve a purpose? I would be worried about relying too much on this -- the word **shouldn't** in here is especially appropriate.

Reply:
I use these for tests, so that I can run the full range of expected values in addition to an unexpected value case